ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -2174,11 +2174,11 @@ FEDataManager::restrictData(const int f_data_idx,
                     F_rhs_e[axis](k) += F_qp * phi_F[k][qp];
                 }
             }
-            for (unsigned int i = 0; i < n_vars; ++i)
+            for (unsigned int var_n = 0; var_n < n_vars; ++var_n)
             {
-                dof_id_scratch = F_dof_indices[i];
-                F_dof_map.constrain_element_vector(F_rhs_e[i], dof_id_scratch);
-                F_rhs_vec->add_vector(F_rhs_e[i], dof_id_scratch);
+                copy_dofs_to_vector(var_n, F_dof_indices, dof_id_scratch);
```

Comment:
👍

Reply:
except ---  sorry to be pedantic ---`copy_dof_ids_to_vector`? these are the DOF indices, not the DOFs themselves . . .

Reply:
Changed!